### PR TITLE
Offers details

### PR DESCRIPTION
**Pull request by jonzee**
Added Accordions with details info + mockup example.

### CODE REVIEW

File: src/ts/Accordion/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <div className="acc-header">
+      <FlatButton
+        icon={icon || <div className="material-icons" />}
+        onTouchTap={call}
+        disableTouchRipple={true}
+      >
+        {head}
+      </FlatButton>
+    </div>);
+}
```
Comment (michalmikolajczyk) [inline src/ts/Accordion/Header.tsx]: looks cool

File: src/ts/Accordion/Table.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+export default function Table({ items }) {
+  const half = Math.ceil(items.length / 2);
+  const table = _.map(_.range(half), index => (
+    <div className="tr" key={index}>
+      <div className="td">
+        <div className="col">{items[index][0]}</div>
```
Comment (michalmikolajczyk) [inline src/ts/Accordion/Table.tsx]: nope

File: src/ts/Accordion/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  render() {
+    const icon = this.state.open
+      ? <FontIcon className="material-icons">expand_less</FontIcon>
+      : <FontIcon className="material-icons">keyboard_arrow_down</FontIcon>;
+    const header = this.header && <Header call={this.toggle} icon={icon} head={this.header} />;
+
+    return (
+      <div className="accordion">
+        {header}
+        {this.state.open ? this.table : null}
```
Comment (michalmikolajczyk) [inline src/ts/Accordion/index.tsx]: {this.state.open && this.table}

File: src/ts/Detail/Form.spec.tsx [hunk 4]
```diff
@@ -3,21 +3,21 @@ import * as React from 'react';
...
 import * as _ from 'lodash';
 import * as moment from 'moment';
-import DetailForm from './DetailForm';
+import Form from './Form';
 import { mountWithTheme } from '../helpers/test-theme';
 import * as offers from './mockup';
 const offer = offers.ktm;
 
-describe('<DetailForm />', () => {
+describe('<Form />', () => {
```
Comment (michalmikolajczyk) [inline src/ts/Detail/Form.spec.tsx]: the name is to generic. Consider a standard prefix, like `vg...` e.g. `vgForm`

File: src/ts/Detail/Form.tsx [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+    super(props);
+    const { offer } = props;
+    this.offer = offer;
+    const price = offer.price.unit[Object.keys(offer.price.unit)[0]];
+    this.state = {
+      price,
+      startDate: moment().toDate(),
+      endDate: moment().add(3, 'days').toDate(),
+      equipment: 1,
+      total: 55 * 3,
```
Comment (michalmikolajczyk) [inline src/ts/Detail/Form.tsx]: surely this should not be the default :joy:

File: src/ts/Detail/Header.spec.tsx [hunk 6]
```diff
@@ -1,14 +1,14 @@
 import 'mocha';
 import * as React from 'react';
 import { expect } from 'chai';
-import DetailHeader from './DetailHeader';
+import Header from './Header';
```
Comment (michalmikolajczyk) [inline src/ts/Detail/Header.spec.tsx]: too generic?

File: src/ts/Detail/Header.tsx [hunk 7]
```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 
-export default function DetailHeader(props) {
-  const { offer } = props;
+export default function Header({ offer }) {
```
Comment (michalmikolajczyk) [inline src/ts/Detail/Header.tsx]: too generic, perhaps you could use a namespace?

File: test-setup.js [hunk 8]
```diff
@@ -29,6 +29,10 @@ global.window.scrollTo = (x, y) => {
   global.window.scrollY = y;
 }
 
+global.document.querySelector = () => ({
```
Comment (michalmikolajczyk) [inline test-setup.js]: woot?

Comment (jonzee) [inline test-setup.js]: it's for `Form` tests in `Detail`

**Pull request merged**